docs/designs/ps_design.md

```diff
@@ -1,377 +1,94 @@
 # Parameter Server Design
+This document describes the design of a distributed parameter server (PS) to support training scalability and PS fault tolerance in ElasticDL.
 
 
-## Overview
-Currently, there is one parameter server (PS) co-existed with the master. In order to support PS with multiple nodes and PS fault tolerance, we need to separate PS from the master. Besides a KV store for model parameters, PS should also support updating parameters using gradients. Currently, models have two kinds of parameter, variable and embedding vector. PS should support model variable update and embedding table sparse update.
+## Motivation
+Parameter server based distributed training uses data parallelism to speed up training. PS stores model parameters. There are multiple workers that get model parameters from PS, compute gradients using different training data and send computed gradients to PS. PS iteratively updates these model parameters using gradients sent by workers. A PS based distributed training can use an arbitrary number of workers to support the scalability of training data size.
```

Comment:
It is not "parameter server-based distributed training uses data parallelism"; instead, it is "data parallelism-based training systems who use data parallelism".

Reply:
We need to reorder these sentences in this paragraph or rewrite them.